server/package.json

```diff
@@ -8,7 +8,9 @@
   },
   "dependencies": {
     "vscode-languageserver": "^2.6.2",
-    "when-switch": "^0.2.0"
+    "when-switch": "^0.2.0",
+    "path": "*",
```

Comment:
Honestly, i don't know a lot about typescript, and it seemed like i needed to do this. Ill remove them if it works fine without (which i assume it does)